witchcraft/internal/tcpjson/tcp_logger_test.go

```diff
@@ -103,7 +103,7 @@ func TestWriteFromSvc1log(t *testing.T) {
 	err = gotPayload.UnmarshalJSON(gotEnvelope.Payload)
 	require.NoError(t, err)
 	assert.Equal(t, "this is a test", gotPayload.Message)
-	assert.Equal(t, logging.LogLevelDebug, gotPayload.Level)
+	assert.Equal(t, logging.New_LogLevel(logging.LogLevel_DEBUG), gotPayload.Level)
```

Comment:
WGS needs WGL to be upgraded to `1.10` in order to write out trace tags properly. There [appears to be a conjure generation change](https://github.com/palantir/witchcraft-go-logging/compare/v1.9.0...v1.10.0) included in 1.10. 

This has meant that the enum names and the structure of the log level are now different. I included these changes here but can do it in a precommit if you prefer.

Reply:
In fact here is that precommit https://github.com/palantir/witchcraft-go-server/pull/266